parameter_optimization/randomforest.py

```diff
@@ -0,0 +1,40 @@
+import sklearn
```

Comment:
instead of importing all of sklearn (which is a ton), just import the parts you need, like you do in the lines below